### ISSUE TITLE
Error loading image

### ISSUE THREAD

**Issue by kevinfealey** [salesforce/salesforce-risk#2]
Receiving the following error when running Initialize() for the first time:

> Exception: Error retrieving image from URL or bad URL: data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAFgAAAAiCAYAAADbLB6TAAAJPElEQVRoge2ae3BU1R3HP/fu3WdCXrvZvB9AEmISYohCAQWjIPhAB6yP0YqjaNWqVcEydupoW+qMHa3D1LFqdbRTWutjfICYVoMoVC0vIQkQBCIhzyUhu5vdZJPs7t272z/uZrNLAAGDgOY7cye7937P73zP9/7O75x7N0LI5eJocIQMSTt2Nt7WsLd9XlNL94XuvsG0oxJ/hJAk0ZeUYOrMyUzZdX5pbnXppPzVVq3ceTSucKTBTsUUX7PxqxWffN54z6GW8SaXPRe3M4ugIn0v4s8FCKKCTjeIKaGb1Mz9WNJs3oVXVfzh4qnlf0oSPP4YbrTBB91yxUt/X/9Ow86MibaDFch+4/cu/lyEztBPfvEXTJ7sq7t/yVXzUiV/99C1iMFtA6GiZ55f+2VDbYXF3jnxjIk9l5Ges4fyyv2tD9199dxsE40QNtipmOKffem97ds2FRY5usbM/S6wZu9h1uyuzffdfu2sJKEvIAGs+++2J/d+nVDU48hDlAJnWuM5DXtnEXX19umbttcvv/LCCU9JNp+Y98nGvQ90ts9B1MhnWt8PAl0dxdRs2Lx0emX5s1LtzqYbHPZ0TTAkjhk8SvD7DRyyxafub25eKO3ae6hqxw4L40wyiQlnWtoPA719sLvBTH1DxxyptcNZZndm0tYuk5kOebkgCGda4unH26smkGFRB7rothbs9u++9vj90NwKdgfEj0ugub35fMndO5jm8wsgyNi6oNcDxYVgMJxaJ0UFBl5ZmQdAY6vCXfd/852FnzaEQgCIgowonrrBoRAc6oKWNlAUQACfT6C13VEuyQHFEAwFQFTJnkGobYCUZEg1q39PJqMFMfqJL4Rw1tb1EKAOTNAETlmnuxe+OQgDA+ETYR9lBfyyYlTdEGODKyHodqqHJEFKEphTwJwMonj8DgWNLupLCEE8Ow0OhqIMFuST0tnnAacLXG7VYCBi7JGQAITjTA8lGGW2Rs1oy5DZmpF8QYiNJWqOP/UEYP4cK5dXWcnPMyGJAl12H//b4uTdD2z09ce2Ly9JYO4lqRQWxGO16NFpRQ7bfWze5uTN9zpG8NNS9dy5OI/K8iQkjcD+Jg+v/bMFjycAFl1Yo3JcnUpANdTuhB43yFH3QviWhFMzWDixuxcIwmGHeoiimtlWi2r4UGbHGhxCOE5sQYDHlpVQ9ZOkCB9CxGdpmXhdGpfOsvDr39fTedgLqH0su7+AHKsmiq+QnyGRf62VaRck88Cj2/H7gwAkJmh59skyMlLESOwLSkxM+m0pTpc/Soc8QmdfHzhcqrF9nki5Djc4IbuAIYNPYRoHAbtLPUQREseByQhJSQHVuZA6oONNvQXzsqmalgihEL1+DdU1bXg8AWbNTKM4V0+2RWTZfUU8umI7oFr01uom5szO5PNNXbS2e9DpNNxxSwGF2TomZEpUXWxm3QYbADctyicjWdXi8Wv4sKYdr09hzuwMcq2aYdfEAH5ZpsetZqjDFZulCJyUqdE4ZYOjEQR6+tQjvTOgFm7AL3vZVCsTZ1TNNxkhzqjuUAx6uHJuViTGyhd38uVW9ZXq2pom/vZcFeZ4qDwvjvxcPS3tHgBqNrRQs6Elpv9Ui8TSJUUA5GUbIzd1xrTU4fgvN7D2ow4GffDK6818+m4Vep3q2ua6AO22Izz4lql/ogjX4NFbiGLreQifLOOTwdkbyzMaBArHx4VpId7/qAOfLxi+KrOl1s5Vl1gBSEuLZ2tdD6IIF55vYcG8HCYVJGJO1mPUS8Tpw+0EAVmBxmaZICI5GcbIFuiZF9sY9CqR+Dt2u5hxQUrk++lajE+qBp8QjljkjhXbYNAihAv3wGCAhkZfzPXWjsHITFCCEnsOyPzq3iKe+s3kCKer28vXB9yMi5MoLlAfQ3v7g7R3yZiTdRFz+zwygz5vzDTvtntjNY+mB1EYlRIRg2iDhdAxY7s8MoFAEEkSMRkljKZgVIaBxayNfHZ7vBiMQZ5Ydl7k3O1LN/P6+80A3HHjBF5+elq4zyCIMv3eYIQbHyeh1SvI8vC55OSo/booj64HURBjOhiVI7ZEHIsXEmS27XREmNfMS49cM8WFuOyi4Z8Av9p1mDSrBqNBNcXd6+f1NY0RfkVZ9EsUBUQZr+yltaMfAEEQuHquNcJPTIQppclRLgRG2YPhGxYuEaP4DlgYzkKr2cDjD5WNoNQ2OPhwfRsv/mMPMyovAeDlP06nvCQRZ4+Pm64ZT0qSHoBtO7up3XMYrVZEUYJoNCKJCTp+fstE6hocXDYzk7tvKYrqPxQZz+qaZh68oxSAV5+eyeTiBnr7/Nx6XQHxcdqoNoHR9SAKo77IRQtNNRt44qEpIyivvb2X6s+aePPDfVTNSGPJjcXEmbQ8em95DK/LPsCS5Z8hiDIBBf71wTcsXqSa+cKTMyO8NeuamT87G4NeQhCCkfE8/dcd3LhgPOmpJhLG6SJanC4vdXvsVJRYVMli4LQtcqe5RBwD4TqJKHPv4+v52dKP2bilA6fLi9+vcKDVzfOr6pm68C32NXdHuA+u2MDzq+pp7/REeL/78xZufvg//DuydRuOfbinl6qb32XNuib6PH76B2TWfdHK3MXvs/K12ig9o18eTCaZlCRTh3DXI6s63/jYmeYPhI7qxRhODeYEkXtuGL9RGp9rrrUkO66w2c/OlzLnKhLG6chIS9wvTa+c8E71xgNX2JwD395qDCeMXGscUyvy10gXTSt4Iy/9y+e+bus19XvHflEeDSSYtGRZTa6ySZmfSnqdNHDt5eUrW7r6Hvuq0X6mtZ3zEASBykIzi6+fvlynkwYlgOsXVK7YWndwkXvAW9J46Oj/DDiGE0NprpmZU/I/u+yiSa9CeB8sSRr/8l/MX+j1Va/TaIJ5+2wOxvYUJ4+yHCtVU/J3P3LP5T8VBCEEIISi3iQ7Xf2Zz7xQs3rL7uap9a0d9Pv8xww2hmEkmYyUZKUzu7Jg4y/vvPRWc3J8+9C1GIMBFCUoVa/f9fDba7f/rsPRE9fR00O/z4d7cJCAoowI/mOEpNFg0GpJT0zEmjCOwqw026IrK56aX1X6l6HMHcIIg4cwMOhP3L3PdunW2oOL7A5PXqvNWebp95m/lxGc5dDrpP6crJTd+dnmuoqynI8qJ+dWS5qjP2v/H21VAbVx8a9GAAAAAElFTkSuQmCC (line 132, file "setup")

Note that in the original code (I made modifications), this was line 131. Lines 133, 209, and 211 generate similar messages. 

This is because the insertImage function with 3 arguments looks for either a URL(https://developers.google.com/apps-script/reference/spreadsheet/sheet#insertimageurl-column-row) or blob (https://developers.google.com/apps-script/reference/spreadsheet/sheet#insertimageblobsource-column-row) in the first parameter. 

I fixed this with the following changes:
```  
# removed the data structure information from the front of the images since we'll provide it as parameters when loading
  var loadImage = "iVBORw0KGgoAAAA...";
  var newReleaseImage = "iVBORw0KGgoA...";
...
 # load the image as a blob
  var imageBlob = Utilities.newBlob(Utilities.base64Decode(loadImage), 'image/png', 'sample');
 # pass in the blob 
 var loadImg = sheet.insertImage(imageBlob, 9,1);
```

**Comment by kevinfealey** [comment]
I don't know how to link a pull request, but the following is intended to resolve this issue: https://github.com/salesforce/salesforce-risk/pull/3

**Comment by jkuemerle** [comment]
Closed #3
